src/config-loader/config-loader.ts

```diff
@@ -195,15 +202,37 @@ export const setup = (app: Express.Application) => {
 						);
 
 						const apiRoute = `/${model.apiRoot}/*`;
+						const webResourceHandler =
+							data.webResourceHandler ?? getDefaultHandler();
+
+						const fileUploadMiddleware =
+							getUploaderMiddlware(webResourceHandler);
 						app
 							.route(apiRoute)
 							.options((_req, res) => res.status(200).end())
 							.get(sbvrUtils.handleODataRequest)
 							.put(sbvrUtils.handleODataRequest)
-							.post(sbvrUtils.handleODataRequest)
-							.patch(sbvrUtils.handleODataRequest)
+							.post(fileUploadMiddleware, sbvrUtils.handleODataRequest)
```

Comment:
@Page- We can add but I think we would be better off doing this in a follow up PR if you don't mind. Mostly because POST/PATCH are the ones that will be consumed right now